seahorn/storage/jobs/ipc/indirect_handlers/CMakeLists.txt

```diff
@@ -1,6 +1,13 @@
-add_executable(storage_ipc_indirect_handlers
-  ${TRUSTY_ROOT}/trusty/user/app/storage/ipc.c
-  indirect_handlers_harness.c)
+if (HANDLE_TYPE_IS_PTR)
```

Comment:
@priyasiddharth you should be able to define a variable that resolves to appropriate `ipc.c`, say called `IPC_C_SOURCE`, set that variable conditionally in a parent `CMakeLists.txt`, and use subsequently without the conditional.

Reply:
Done